libs/orders/src/lib/order-hooks/use-order-validation.spec.tsx

```diff
@@ -90,107 +90,146 @@ function setup(
   return renderHook(() => useOrderValidation({ ...defaultOrder, ...props }));
 }
 
-it('Returns empty string when given valid data', () => {
-  const { result } = setup();
-  expect(result.current).toEqual('');
-});
-
-it('Returns an error message when no keypair found', async () => {
-  const { result } = setup(defaultOrder, { keypair: null });
-  expect(result.current).toEqual('');
-});
+describe(`useOrderValidation`, () => {
+  it('Returns empty string when given valid data', () => {
+    const { result } = setup();
+    expect(result.current).toStrictEqual({ isDisabled: false, message: `` });
+  });
 
-it('Returns an error message when the keypair is tainted', async () => {
-  const { result } = setup(defaultOrder, {
-    keypair: { ...defaultWalletContext.keypair, tainted: true },
+  it('Returns an error message when no keypair found', async () => {
```

Comment:
question (blocking): Do all these tests need to be async. Nothing async is happening in the hook or the tests

Reply:
I did not change this, they were async before but they don't need to be. Updated.